packages/ag-grid-theme/css/_uitk-ag-theme-icon-font.scss

```diff
@@ -17,7 +17,6 @@ $icons: (
   eye: "\e965",
   filter: "\e922",
   first: "\e934",
-  group: "\e929",
```

Comment:
deleting this as its incorrect anyway, and we are actually removing the display in some cases in packages/ag-grid-theme/css/_uitk-ag-theme-mixin.scss  line 299. since we didnt have an icon, best to fall back to ag grid default if it does appear